src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="product-card" data-qa="card">
+      <div class="product-card__image">
+        <img
+          src="images/imac.jpeg"
+          class="product-card__imac"
+          alt="imac"
+          >
+      </div>
+      <div class="product-card__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="product-card__code">
+        Код товара: 195434
+      </div>
+      <div class="product-card__reviews">
+        <div class="product-card__stars">
+          <div class="product-card__onestar"></div>
+          <div class="product-card__onestar"></div>
+          <div class="product-card__onestar"></div>
+          <div class="product-card__onestar"></div>
+          <div class="product-card__onestar"></div>
+        </div>
```

Comment:
this block should work the same way, as in the `stars` task. modifier --3/--4 should change count of active stars